### ISSUE TITLE
Preprocessing before simulations

### ISSUE THREAD

**Issue by EylonSho** [TechnionYearlyProject/SummitBTW#52]
Before starting a simulation we have to prepare a few things and get the following ready:
* Convert an OSM map into SUMO network (currently it has low priority, since we can use ready SUMO networks).
* Adding detectors to for each traffic light - #39, #40. (I'm in charge)
* Adding logic for each traffic lights - valid states and order (@yairf11 is in charge)
* Generating random trips and routs for vehicles in the simulation. (@LiranFarhi is in charge).
  * Targeting for customising traffic on specific junctions (#41).

**Comment by tomerkay** [comment]
@gdor80 and I are working on converting an OSM map into SUMO network.

**Comment by gdor80** [comment]
@tomerkay and I found [NETEDIT](http://sumo.dlr.de/wiki/NETEDIT) which lets us import sumo net files and graphically edit anything about it, particularly interesting is the ability to edit traffic lights static scheduling and junction connection information.

**Comment by EylonSho** [comment]
Currently we are not focusing on OSM conversion to SUMO network.
After the success to automatically generate SUMO network with traffic lights included - as described in #78, we can say that the goals that were described above are achieved (or not relevant currently).
Closing.
